src/components/workspace-utils.js

```diff
@@ -378,7 +378,7 @@ export const WorkspaceStarControl = ({ workspace, starredWorkspaceIds, onUpdate,
     as: 'span',
     role: 'checkbox',
     'aria-checked': isStarred,
-    tooltip: isStarred ? 'Unstar this workspace' : 'Star this workspace',
+    tooltip: isStarred ? 'Unstar this workspace' : 'Star this workspace. Starred workspace will appear at the top of your workspace list',
```

Comment:
Can you add a period at the end of the sentence?

Reply:
Done in 740f866cc7b1bbea5687bd00973264508f053d44